pkgs/development/libraries/gettext/gettext-setup-hook.sh

```diff
@@ -5,3 +5,13 @@ gettextDataDirsHook() {
 }
 
 addEnvHooks "$hostOffset" gettextDataDirsHook
+
+# libintl must be listed in load flags on non-Glibc
+# it doesn't hurt to have it in Glibc either though
+gettextLdflags() {
+    export NIX_LDFLAGS="$NIX_LDFLAGS -lintl"
```

Comment:
We'll need to case on `$hostOffset` to use the correct one of `NIX_{BUILD_,_,TARGET_}LDFLAGS`. See cc-wrapper for details.

Reply:
Would that be something like "NIX_@infixSalt@_LDFLAGS"?

Reply:
Does this still need to be changed? @Ericson2314

Reply:
Or at least does merging as is break anything?

Reply:
Oh bummer I forgot about this. Yes this should be fixed.

Reply:
I'll go do this.

Reply:
https://github.com/NixOS/nixpkgs/pull/37707